### ISSUE TITLE
Basic usage examples failing

### ISSUE THREAD

**Issue by bmorris3** [gaia-dpci/GaiaXPy#65]
I've installed GaiaXPy, both via pip and from source, and run:
```python
from gaiaxpy import generic_function
```
as [the docs recommend](https://github.com/gaia-dpci/GaiaXPy/blob/main/docs/usage.rst#input). I'm getting the following error:
```
ImportError: cannot import name 'generic_function' from 'gaiaxpy' (/Users/bmmorris/miniconda3/envs/research3/lib/python3.10/site-packages/gaiaxpy/__init__.py)
```

**Desktop:**
 - OS: Mac OS X
 - Browser Chrome
 - Version 2.1.0

**Comment by druzm** [comment]
Hi, `generic_function` is used as a placeholder for any of the main functions in the package (`calibrate`, `convert`, etc.) As most functions shared many of the same parameters, the use of `generic_function` avoids duplication in the documentation. There's a note about this right above the section of the page your link points to (no worries, it happens.)

You can find more detailed tutorials on this [link](https://gaia-dpci.github.io/GaiaXPy-website/tutorials.html).

**Comment by druzm** [comment]
Yes. We added the note to try to clarify this point, but I understand it may be confusing if you're just trying the package and trying to run some basic code just to see it work. At the same time, adding the same instructions for every function would be unnecessarily long.

At least in the [official docs](https://gaiaxpy.readthedocs.io/en/latest/usage.html#generic-usage) and not in the RST file, the note is more visible.

Let me know if you get any idea of how to avoid this problem. This is the best solution we could find.
